examples/ucp_hello_world.c

```diff
@@ -603,17 +600,22 @@ int main(int argc, char **argv)
         oob_sock = connect_common(NULL, server_port, ai_family);
         CHKERR_JUMP(oob_sock < 0, "server_connect\n", err_peer_addr);
 
-        addr_len = local_addr_len;
+        addr_len = worker_attr.address_length;
+
         ret = send(oob_sock, &addr_len, sizeof(addr_len), 0);
         CHKERR_JUMP_RETVAL(ret != (int)sizeof(addr_len),
                            "send address length\n", err_peer_addr, ret);
 
-        ret = send(oob_sock, local_addr, local_addr_len, 0);
-        CHKERR_JUMP_RETVAL(ret != (int)local_addr_len, "send address\n",
+        ret = send(oob_sock, worker_attr.address, addr_len, 0);
```

Comment:
minor: `addr_len` seems redundant here - IMO `worker_attr.address, worker_attr.address_length` looks clearer

Reply:
So the all this "OOB connection establishment" part would look like the following:
```c
        ret = send(oob_sock, &worker_attr.address_length,
                   sizeof(worker_attr.address_length), 0);
        CHKERR_JUMP_RETVAL(ret != (int)sizeof(worker_attr.address_length),
                           "send address length\n", err_peer_addr, ret);

        ret = send(oob_sock, worker_attr.address, worker_attr.address_length,
                   0);
        CHKERR_JUMP_RETVAL(ret != (int)worker_attr.address_length,
                           "send address\n", err_peer_addr, ret);
```
Instead of current:
```c
        addr_len = worker_attr.address_length;

        ret = send(oob_sock, &addr_len, sizeof(addr_len), 0);
        CHKERR_JUMP_RETVAL(ret != (int)sizeof(addr_len),
                           "send address length\n", err_peer_addr, ret);

        ret = send(oob_sock, worker_attr.address, addr_len, 0);
        CHKERR_JUMP_RETVAL(ret != (int)addr_len, "send address\n",
                           err_peer_addr, ret);
```
As you can see the first option contains many line breaks in the middle of the function and at my point of view it looks more complex that the second one. So I would prefer to keep the things in current shape.

Reply:
Then you can add another variable `addr` to save couple lines. My comment was about making func parameters aligned.